tcpclient/Client.java

```diff
@@ -0,0 +1,129 @@
+
+import java.net.*;
+import java.io.*;
+import java.util.*;
+/**
+ * @author Pontus Laestadius
+ * Date format: DD-MM-YYYY
+ * @since 20-03-2017
+ * Maintained since: 17-04-2017
+ */
+
+public class Client {
+
+	static int port = 9005;
+	static String host = "192.168.0.120";
+	static Boolean c = false;
+
+	public static Transmitter init(String host, int port){
+		return new Transmitter(host, port);
+	}
+}
+
+class BaseSocket implements Runnable {
+	private String host;
+	private int port;
+	DataOutputStream out;
+	BufferedReader in;
+
+	// A queue is used to handle all input commands so they go in the proper order and are not lost.
+	Queue<String> input = new PriorityQueue<>();
+	Queue<String> output = new PriorityQueue<>(); // TODO: 06/04/2017 make a read function for this.
+	Socket socket;
+
+	// Macro for add.
+	public void write(String s){
+		// TODO: 06/03/2017 Add command vertification here. Preferably O(1).
+
+		try{
+			if (System.currentTimeMillis() %30 != 0)
+				out.writeUTF(s + "\n");
+			out.flush();
+		} catch (Exception e){
+			e.printStackTrace();
+		}
+
+	}
+
+	/**
+	 *
+	 * @return the first command in the queue.
+	 */
+	public String read(){
+
+		try {
+			in = new BufferedReader(new InputStreamReader(socket.getInputStream())); // I hate Java.
+			return in.readLine();
+			
+		} catch (IOException e){
+			e.printStackTrace();
+		} finally {
+			return ""; // Only occurs if an exception is thrown.
+		}
+	}
+
+	/**
+	 *
+	 * @return all the queued up output as an array of strings.
+	 */
+
+    /*
+    public String[] readAll(){
+        int i = 0;
+        String[] res = new String[output.size()];
+        while (!output.isEmpty())
+            res[i++] = read();
+        return res;
+    }
+    */
+
+	/**
+	 *
+	 * @return a formated version of all queued up output received.
+	 */
+    /*
+    public String readAllFormated(){
+        String[] format = readAll();
+        String formatted = "";
+        for (String f: format)
+            formatted += f + ", ";
+        return formatted.substring(0, formatted.length()-3);
+    }
+*/
```

Comment:
It's for a functionality not currently used after I altered the Android application to not require these.